### ISSUE TITLE
Question about the Table 2 and 3, i.e., results on the well-known datasets and with standard split

### ISSUE THREAD

**Issue by rllm-project** [chennnM/GCNII#4]
The results seems to be very good, cons.
We have re-run the code and get the similar results **with fixing the seed=42**.
We wish to cite this work with reporting the given results on these three standard datasets.
Before that, there are some questions:

1. The reported results in Table 2 are just with the fixed seed? or with random seeds and run many times?
2. If you choose the best model (with hyper-parameters) based  on the validation set in the experiments in Table 2, what's the setting of Table 3, like fixing the seed or others? 

thanks,

**Comment by chennnM** [comment]
Hi, thanks for your attention.
1. Table 2 reports the average accuracy after 100 runs with a random seed.
2. The setting and hyperparameters in Table 3 are the same as those in Table 2.

**Comment by rllm-project** [comment]
thx
